config/sys.config

```diff
@@ -26,7 +26,8 @@
         {random_peer_pred, fun bn_sup:random_val_predicate/1}
     ]},
     {blockchain, [
-        {s3_base_url, "https://snapshots.helium.wtf/mainnet"},
+        {snap_source_base_url, "https://snapshots.helium.wtf/mainnet"},
+        {fetch_latest_from_snap_source, true},
```

Comment:
Remember that we did _not_ want to do this to avoid gaps in blockchain_node restarts that have been down a number of days and then restarted, _or_ restarted from a backup

Reply:
roger. mark as false?